src/r_plane.c

```diff
@@ -3,6 +3,8 @@
 //  id Software, Chi Hoang, Lee Killough, Jim Flynn, Rand Phares, Ty Halderman
 //  Copyright (C) 2006-2025 by
 //  The Odamex Team.
+//  Copyright (C) 2013-2025 by
```

Comment:
I don't know why you add these line breaks?

Reply:
These added copyright lines in each file should be dropped in my opinion. The readme.md line addition is sufficient.